Hadoop/Hadoop_Draft_one.md

```diff
@@ -0,0 +1,54 @@
+![Hadoop Art](Assets/hadoop-art.jpg)
+
+***"In pioneer days they used oxen for heavy pulling, and when one onx couldn't budge a log, they didn't try to grow a larger ox. We shouldn't be trying for bigger computers, but for more systems of computers." - Grace Hopper***
+
+In a world we are looking for results in minutes instead of hours or days instead of months, our reliance on single computer networks for data processing can be naive. In this new age, it is better to divide the workload between computers, both in terms of time and cost.
+
+If the reader has past history with computers, the idea of computer and network topologies is not new. In fact, they have been a prominent part of our network configurations (think back to Local Area Network (LAN) and Wide Area Networks (WAN) ). Although these ideas are a part of our daily life. They are limited in what the do. They provide simple network services which users use independently. But we are living in an age of Data and the amount of data flooded in the world every day is estimated, for 2020, to be around 44 zettabytes (4.4*10^22 bytes). In terms of magnitude, this is more than one disk drive for every person in the world.

```

Comment:
I'd add a tera/gigabyte equivalent here, since disk drives are so variable. using some weird baseline measurement examples I've seen before might also help understand how big this number is. something like: over  44 billion terabytes, or enough data to store all the movies from the mcu together over a **trillion** times